amp-sql/Dockerfile

```diff
@@ -24,29 +24,15 @@
 
 FROM postgres:14
 
-COPY postgres/Database_Scripts/Tables/amp_core_create_database.sql /docker-entrypoint-initdb.d/00-amp_core_create_database.sql
-COPY postgres/Database_Scripts/Tables/transcoder.sql /docker-entrypoint-initdb.d/00-transcoder.sql
-COPY postgres/Database_Scripts/Tables/adm_data.sql /docker-entrypoint-initdb.d/01-adm_data.sql
+# COPY postgres/Database_Scripts/Tables/*.sql /docker-entrypoint-initdb.d/
```

Comment:
If not needed, can commented source be removed?

Reply:
This was just for renaming and cleaning up the old dockerfiles.